index.bs

```diff
@@ -551,25 +597,32 @@ spec:css-syntax-3;
 
   <div algorithm="relevant credential interfaces">
     The <dfn for="CredentialRequestOptions">relevant credential interface objects</dfn> for a given
-    {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
+    {{CredentialRequestOptions}} or {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
 
     <ol class="algorithm">
-      1.  Let |settings| be the <a>current settings object</a>
+        1.  Let |settings| be the <a>current settings object</a>
+
+        1.  Let |interface objects| be the set of [=interface objects=] on |settings|'
+            [=environment settings object/global object=].
+
+        1.  Let |relevant interface objects| be an [=set/empty=] [=set=].
+
+        1.  For each |object| in |interface objects|:
+
+            1.  If |object| is not a {{Credential}} interface object, and |object|'s [=inherited interfaces=]
+                do not <a for="set">contain</a> {{Credential}}, then
+                <a for="iteration">continue</a>.
 
-      2.  Let |interface objects| be the set of [=interface objects=] on |settings|'
-          [=environment settings object/global object=].
-         
-      3.  Let |relevant interface objects| be an empty set.
-      
-      4.  For each |object| in |interface objects|:
+            1.  Let |credentialType| be |object|'s {{Credential/[[type]]}} slot's value.
 
-          1.  If |object|'s [=inherited interfaces=] do not <a for="set">contain</a> {{Credential}},
-              <a for="iteration">continue</a>.
+            1.  Let |key| be the result of looking up the |credentialType|'s |options|'s
+                [=dictionary member|member=] [=identifier=] in the registry of
+                [[#sctn-registry-cred-type-to-options-member-identifier|Credential Type to Options Member Identifier Mappings]].
 
-          2.  Let |key| be |object|'s {{Credential/[[type]]}} slot's value.
+            1.  If |options|[|key|] <a for="map">exists</a>, <a for="set">append</a> |object| to
+                |relevant interface objects|.
 
-          3.  If |options|[|key|] <a for="map">exists</a>, <a for="set">append</a> |object| to
-              |relevant interface objects|.
+        1.  Return |relevant interface objects|.
     </ol>
 
     ISSUE: jyasskin@ suggests replacing the iteration through the interface objects with a registry.
```

Comment:
Remove this note

Reply:
Well, I think that issue remains relevant in that the registry does not obviate "iterating through the interface objects"---which is what jyasskin was apparently suggesting---the registry is simply mapping cred types to options' member identifiers. I.e., we still need to discover the cred [[type]] value in play.  Changing the latter discovery to be accomplished somehow using a registry seems to me to be separate from solving the "public-key" vs "publicKey" problem that this PR is addressing.  I've added a parenthetical note to the effect that the issue is not obviated by the registry.
done in eed4835

Reply:
see also https://github.com/w3c/webappsec-credential-management/pull/181/files#r758644349 --- once the latter discussion is resolved we may be able to remove this inline issue.